cmd/end2endtest/zkweighted.go

```diff
@@ -163,8 +167,64 @@ func (t *E2EAnonElectionTempSIKs) Run() error {
 		return err
 	}
 
+	// the half of accounts should not have valid sik, due they are using temporal sik
+	halfAccts := len(votes) / 2
```

Comment:
the name `halfAccts` sounds strange to me, as if it held accounts, but actually is just an int. it's used just in two places and the len() / 2 is very cheap so i'd just remove this var and just put `len(votes)/2` in both places where it's used, i think it's clearer

same with `accs`, in that case i'd call it `count` or `removedCount` (also in the returned args of the func)